src/main/java/com/tabnine/capabilities/Capability.java

```diff
@@ -7,4 +7,7 @@ public enum Capability {
   INLINE_SUGGESTIONS,
   @SerializedName("alpha")
   ALPHA,
+
+  @SerializedName("suggestion_hint_enabled")
```

Comment:
first_suggestion_hint?

Reply:
im worried that we'll have a feature which includes suggestion hints, but unrelated to this feature at all - then we'll have a bit of a hard time with the naming convention. 
i'll leave it to your decision, not critical 🙂

Reply:
yeah, that could happen. yalla, I'm changing it. :)